src/components/SubStepForms/ConfirmationStep.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import DotIndicatorMessage from '@components/DotIndicatorMessage';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import SafeAreaConsumer from '@components/SafeAreaConsumer';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+
+type SummaryItem = {
+    description: string;
+    title: string;
+    shouldShowRightIcon: boolean;
+    onPress: () => void;
+};
+
+type ConfirmationStepProps = SubStepProps & {
+    /** The title of the step */
+    pageTitle: string;
+
+    /** The summary items to display */
+    summaryItems: SummaryItem[];
+
+    /** Whether show additional section with Onfido terms etc. */
+    showOnfidoLinks: boolean;
+
+    /** The title of the Onfido section */
+    onfidoLinksTitle?: string;
+
+    /** Whether the data is loading */
+    isLoading?: boolean;
+
+    /** The error message to display */
+    error?: string;
+};
+
+function ConfirmationStep({pageTitle, summaryItems, showOnfidoLinks, onfidoLinksTitle, isLoading, error, onNext}: ConfirmationStepProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {isOffline} = useNetwork();
+
+    return (
+        <SafeAreaConsumer>
+            {({safeAreaPaddingBottomStyle}) => (
+                <ScrollView
+                    style={styles.pt0}
+                    contentContainerStyle={[styles.flexGrow1, safeAreaPaddingBottomStyle]}
+                >
+                    <Text style={[styles.textHeadlineLineHeightXXL, styles.ph5, styles.mb3]}>{pageTitle}</Text>
+                    {summaryItems.map(({description, title, shouldShowRightIcon, onPress}) => (
+                        <MenuItemWithTopDescription
+                            description={description}
```

Comment:
I think we should add a `key` props here.

Reply:
good point, thanks - added missing `key` prop